pymeasure/display/widgets/plot_widget.py

```diff
@@ -41,20 +41,21 @@ class PlotWidget(TabWidget, QtWidgets.QWidget):
     """
 
     def __init__(self, name, columns, x_axis=None, y_axis=None, refresh_time=0.2,
-                 check_status=True, linewidth=1, parent=None):
```

Comment:
Refer to my comment regarding `setup=False` in multiplot_widget.py: I think the changes in this file are not necessary.

Reply:
Reverted this change